components/Layout/index.js

```diff
@@ -0,0 +1,26 @@
+import Footer from "../Footer";
+import Header from "../Header";
+import styled from "styled-components";
+
+export default function Layout({ children }) {
+  return (
+    <>
+      <Header></Header>
```

Comment:
same here

Reply:
Hi Feline, i would complete the layout. The header will be build in US 9 :-)